consensus/src/checkpoint.rs

```diff
@@ -69,3 +69,89 @@ pub fn is_short_range(
         Ok(false)
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
@hanabi1224 moved these tests to the tests folder see [link ](https://github.com/ChainSafe/mina-rs/pull/90/files)will these be fixed automatically? If not please help delete tests here.